.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -12,17 +12,23 @@ body:
           - Posting on [Discourse](https://discuss.python.org/c/users/7)
           - Reading the [Python tutorial](https://docs.python.org/3/tutorial/)
           - Emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
-  - type: checkboxes
+
```

Comment:
https://stackoverflow.com/ is also good place for people to get help.

Reply:
haven't heard of that one